content/_code-samples/airgapped-wallet/py/README.md

```diff
@@ -0,0 +1,113 @@
+
+Do not reconnect the airgapped machine to a network, even when you need to transfer files! An effective airgapped machine should only serve 1 purpose, which is to store data and never open up a gateway for hackers to abuse and steal data.
+
+# Tutorial
+For testing purposes, you would need to have 2 machines and 1 phone in hand to scan the QR code.
+
+1. 1st machine would be airgapped, following the security practices written [here](#security-practices). It stores and manages an XRPL Wallet. 
+2. 2nd machine would be a normal computer connected to the internet. It relays a signed transaction blob to a rippled node.
+3. The phone would be used to scan a QR code, which contains a signed transaction blob. The phone would transmit it to the 2nd machine.
+
+The diagram below shows you the process of submitting a transaction to the XRPL:
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/87929946/197970678-2a1b7f7e-d91e-424e-915e-5ba7d34689cc.png" width=75% height=75%>
+</p>
+
+# Setup
+- Machine 1 - An airgapped computer (during setup, it must be connected to the internet to download the files)
+- Machine 2 - A normal computer connected to the internet
+- Phone - A normal phone with a working camera to scan a QR
+
+## Machine 1 Setup
+Since this machine will be airgapped, it is best to use Linux as the Operating System.
+
+1. Install Python 3.8:
+
+**Linux Command Line**:
+```
+sudo apt-get update
+sudo apt-get install python3.8 python3-pip
+```
+**Website**: https://www.python.org/downloads/source/
+
+2. Clone all the files under the [`airgapped-wallet`](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/airgapped-wallet/py) directory
+
+3. Import all the modules required by running:
+```
+pip install -r requirements.txt
+```
+
+4. Airgap the machine by following the security practices written [here](#security-practices).
+
+5. Run `airgapped-wallet.py`
+
+6. Scan the QR code and fund the account using the [testnet faucet](https://test.bithomp.com/faucet/) 
+
+## Machine 2 Setup
+This machine will be used to transmit a signed transaction blob from Machine 1, it would require internet access.
+
+1. Install Python 3.8
+
+**Linux Command Line**:
+```
+sudo apt-get update
+sudo apt-get install python3.8 python3-pip
+```
+**Website**: https://www.python.org/downloads/source/
+
+2. Clone all the files under the [`airgapped-wallet`](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/airgapped-wallet/py) directory
+
+3. Import all the modules required by running:
+```
+pip install -r requirements.txt
+```
+
+4. Edit line 47 @ `relay-transaction.py` and insert the signed transaction blob
+
+5. Run `relay-transaction.py`
+
+## Phone Setup
```

Comment:
I'd recommend removing this section, and just having the instructions be part of Machine 1 setup (Ex. of how that could be done in the above recommendation)

Reply:
I have to disagree here, I'm sure that there will definitely be users who aren't familiar with this.